lib/ex_unit/lib/ex_unit/callbacks.ex

```diff
@@ -234,28 +234,33 @@ defmodule ExUnit.Callbacks do
   end
 
   @doc false
-  def __merge__(_mod, context, :ok) do
+  def __merge__(mod, context, value) do
+    __merge__(mod, context, value, value)
+  end
+
+  @doc false
+  def __merge__(_mod, context, :ok, _original_value) do
```

Comment:
You're absolutely right. I was looking at the wrong line. I've updated `__merge__/4` to be a private function and amended it in 2c2b68c20e0768c275a043b650b5101e89a11719. Thanks!